.github/mergify.yml

```diff
@@ -68,3 +68,63 @@ pull_request_rules:
     merge:
       method: merge
     delete_head_branch:
+
+- name: label-cicd
+  description: Automatically apply CI/CD label
+  conditions:
+    - or:
+      - files=.github/labeler.yml
+      - files=.github/mergify.yml
+      - files=.github/workflows/**/*
+      - files=scripts/ruff.sh
+      - files=.pre-commit-config.yaml
+      - files=.pylintrc
+      - files=.spellcheck-en-custom.txt
+      - files=.spellcheck.yml
+      - files=tox.ini
+  actions:
```

Comment:
Should we add https://github.com/instructlab/instructlab/blob/main/.markdownlint-cli2.yaml to this this?